src/main/java/net/sf/jabref/model/entry/BibLatexEntryTypes.java

```diff
@@ -871,4 +871,10 @@ public String getName() {
     public static Optional<EntryType> getType(String name) {
         return ALL.stream().filter(e -> e.getName().equalsIgnoreCase(name)).findFirst();
     }
+
+    public static final List<String> BIBLATEX_PERSON_NAME_FIELDS = Arrays.asList("author", "editor", "editora",
```

Comment:
I was considering that and am open to input here. I can see the point of keeping model clean, but on the other hand, it seems to make sense to keep the information "close to" the source. (In IEEETranEntryTypes the corresponding fields are there.)